openfe/protocols/openmm_rfe/_rfe_utils/compute.py

```diff
@@ -50,7 +50,7 @@ def get_openmm_platform(platform_name=None):
         platform.setPropertyDefaultValue(
                 'DeterministicForces', 'true')
 
-    if name != 'CUDA':
```

Comment:
Do we need to set DeterministicForces or Percision for HIP?

Reply:
Good catch, I was able to do that with the OpenMM API, but I need to test on a real GPU to see if it works (as in, setting those properties worked on my openMM HIP build but it could crash as soon as I try and run a MD step on the GPU)

Reply:
Is there an easy way to test this out? I.e. would spinning up an AWS runner that uses an AMD card work?